src/RegisterMappedEventSubscribers.php

```diff
@@ -30,11 +30,11 @@ class RegisterMappedEventSubscribers implements DoctrineExtender
     protected $container;
 
     /**
-     * @param Container $container
+     * RegisterMappedEventSubscribers constructor.
      */
-    public function __construct(Container $container)
+    public function __construct()
     {
-        $this->container = $container;
+        $this->container = Container::getInstance();
```

Comment:
Perhaps it's easier to change ` $this->container['config']` to -> `app('config')`

Reply:
I'll try to fix it and check it out soon.